include/z64save.h

```diff
@@ -557,6 +557,20 @@ typedef enum {
 #define HS_GET_SWAMP_SHOOTING_GALLERY_HIGH_SCORE() ((s32)((HIGH_SCORE(HS_SHOOTING_GALLERY) & 0xFFFF0000) >> 0x10))
 #define HS_SET_SWAMP_SHOOTING_GALLERY_HIGH_SCORE(score) (HIGH_SCORE(HS_SHOOTING_GALLERY) = (HIGH_SCORE(HS_SHOOTING_GALLERY) & 0xFFFF) | ((u16)(score) << 0x10))
 
+#define ALIEN_GET_SPAWN_TIME_OFFSET(index) \
+    ((index % 2) == 0) ? ((gSaveContext.save.saveInfo.alienInfo[index >> 1] & 0xFFFF)) \
+                       : ((gSaveContext.save.saveInfo.alienInfo[index >> 1] & ~0xFFFF) >> 0x10)
+
+#define ALIEN_SET_SPAWN_TIME_OFFSET(index, spawnTime) \
+    ((index % 2) == 0) ? (gSaveContext.save.saveInfo.alienInfo[index >> 1] = (gSaveContext.save.saveInfo.alienInfo[index >> 1] & ~0xFFFF) | (spawnTime & 0xFFFF)) \
+                       : (gSaveContext.save.saveInfo.alienInfo[index >> 1] = (gSaveContext.save.saveInfo.alienInfo[index >> 1] & 0xFFFF) | ((spawnTime & 0xFFFF) << 0x10))
```

Comment:
Yeah, in practice, the `index` supplied here is just 0-7, which exactly corresponds to the indices used by EnInvadepoh. In theory, though, you could *also* use indices 10-13 too; you just have to avoid indices 8 and 9 to avoid overwriting the alien count stuff.

Should I put this in a comment above these macros? I'm not sure if that's getting too into the weeds about this.

Reply:
Oh, I didn't notice 10-13 are available too. I guess you could mention them being available but unused.